.eslintignore

```diff
@@ -1 +1,2 @@
 tmp/
+test/
```

Comment:
Shouldn't the test script still be linted?

Reply:
ESLint can't parse the test file because it uses `import.meta`. We'd need to switch to babel-eslint parser or find a way to avoid `import.meta`. Refs: https://github.com/eslint/eslint/issues/11189#issuecomment-446666614

Reply:
I put that information in a comment. Optimistically marking this as "resolved" but if you think that's not satisfactory, please unresolve!